samples/Databricks-AzureSQL/DatabricksNotebooks/SQL Spark Connector - AAD Auth.scala

```diff
@@ -0,0 +1,55 @@
+// Databricks notebook source
+// DBTITLE 1,Import Microsoft ADAL Library
+// Provides Access Token
+import com.microsoft.aad.adal4j.{AuthenticationContext, ClientCredential}
+
+// COMMAND ----------
+
+// DBTITLE 1,Import Dependencies
+import org.apache.spark.sql.SparkSession
```

Comment:
We will need a PySpark version of this sample to fit in with the rest of the samples, which are predominantly Python based. Is that something you can possibly work on, @cchighman ?

Reply:
Yes

Reply:
PySpark examples have been added for both Service Principal and Active Directory Password.